passes/cmds/viz.cc

```diff
@@ -892,7 +892,7 @@ struct VizPass : public Pass {
 		log_header(design, "Generating Graphviz representation of design.\n");
 		log_push();
 
-#if defined(_WIN32) || defined(YOSYS_DISABLE_SPAWN)
```

Comment:
Same as above, don't remove this one conditional

Reply:
Fixed